packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -57,6 +57,14 @@ class _RenderExclusiveMouseRegion extends RenderMouseRegion {
   static bool isOutermostMouseRegion = true;
   static bool foundInnermostMouseRegion = false;
 
+  @override
+  void handleEvent(PointerEvent event, HitTestEntry entry) {
```

Comment:
If the hit test behavior is deferring to the child, assuming the scroll view handles the event, why does it need to be handled here?